<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContentViewWidget</class>
 <widget class="QWidget" name="ContentViewWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>496</height>
   </rect>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="KTabWidget" name="mainTabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Payload</string>
      </attribute>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QStackedWidget" name="stack">
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="currentIndex">
          <number>3</number>
         </property>
         <widget class="QWidget" name="unsupportedTypePage">
          <layout class="QVBoxLayout">
           <item>
            <widget class="QLabel" name="unsupportedType">
             <property name="text">
              <string>Unsupported Type</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="addresseeViewPage">
          <layout class="QVBoxLayout">
           <item>
            <widget class="KPIM::AddresseeView" name="addresseeView"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="incidenceViewPage">
          <layout class="QVBoxLayout">
           <item>
            <widget class="Akonadi::KCalItemBrowser" name="incidenceView"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="mailViewPage">
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="KMReaderWin" name="mailView"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Attributes</string>
      </attribute>
      <layout class="QGridLayout">
       <item row="0" column="0" colspan="3">
        <widget class="QTreeView" name="attrView">
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="KLineEdit" name="attrName">
         <property name="clickMessage">
          <string>New Attribute</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="KPushButton" name="attrAddButton">
         <property name="text">
          <string>&amp;Add</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="KPushButton" name="attrDeleteButton">
         <property name="text">
          <string>&amp;Delete</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Raw Payload</string>
      </attribute>
      <layout class="QGridLayout">
       <item row="0" column="0" colspan="2">
        <widget class="QTextEdit" name="dataView">
         <property name="lineWrapMode">
          <enum>QTextEdit::NoWrap</enum>
         </property>
         <property name="acceptRichText">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Internals</string>
      </attribute>
      <layout class="QGridLayout">
       <item row="0" column="0">
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Unique ID:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="id">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Remote ID:</string>
           </property>
           <property name="buddy">
            <cstring>remoteId</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KLineEdit" name="remoteId"/>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Revision:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="revision">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Size:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="size">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Last modification time:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="modificationtime">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="0">
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Mimetype:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="mimeType">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>invalid/invalid</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="6" column="0">
        <widget class="KEditListBox" name="flags">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Flags</string>
         </property>
         <property name="buttons">
          <set>KEditListBox::Add|KEditListBox::Remove</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="nepomukTab">
      <attribute name="title">
       <string>Nepomuk</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTreeView" name="nepomukView">
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="allColumnsShowFocus">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="KPushButton" name="saveButton">
     <property name="text">
      <string>&amp;Save</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KEditListBox</class>
   <extends>QGroupBox</extends>
   <header>keditlistbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KTabWidget</class>
   <extends>QTabWidget</extends>
   <header>ktabwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KTextBrowser</class>
   <extends>QTextBrowser</extends>
   <header>ktextbrowser.h</header>
  </customwidget>
  <customwidget>
   <class>KPIM::AddresseeView</class>
   <extends>KTextBrowser</extends>
   <header location="global">libkdepim-copy/addresseeview.h</header>
  </customwidget>
  <customwidget>
   <class>Akonadi::KCalItemBrowser</class>
   <extends>KTextBrowser</extends>
   <header location="global">kcal/kcalitembrowser.h</header>
  </customwidget>
  <customwidget>
   <class>KMReaderWin</class>
   <extends>KTextBrowser</extends>
   <header location="global">libmailreader/kmreaderwin.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
